### PR TITLE
Subversion (svn) cheatsheet

### PR DESCRIPTION
**Pull request by Juholei**
**What does your Instant Answer do?**
My Cheat sheet displays some common commands for Subversion, or svn, version control system.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Data source is the book "Version Control with Subversion" that is available at http://svnbook.red-bean.com/.

**Why did you choose this data source?**
It is the guide to Subversion that is promoted at https://subversion.apache.org/docs/ as the user manual for Subversion.

**Are there any other alternative (better) data sources?**
There could be ones that are clearer, but I chose this one as it is the nearest to official documentation.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Software developers that use Subversion.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
I submitted the idea there: [SVN cheatsheet](https://duck.co/ideas/idea/5998/svn-cheatsheet)

**Are you having any problems? Do you need our help with anything?**
There weren't any problems while implementing this. As there was no approval in Instant Answer Ideas though, I didn't at first know whether I should start making this. As it didn't take long, I thought to do anyway just to learn new things.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Blogs like [Marco.org](http://www.marco.org/) and [Daring Fireball](https://daringfireball.net/) introduced me to DuckDuckGo. I started following DuckDuckGo in Twitter and saw DuckDuckHack there.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot-svn-cheatsheet

-----
IA Page: https://duck.co/ia/view/svn_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/svn.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+    "id": "svn_cheat_sheet", 
+    "name": "Subversion",
+    "description": "Subversion help",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/svn.json]: Let's remove this, or make it a little more descriptive about what the cheat sheet provides

**Comment by moollaza** [comment]
@Juholei thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

**Comment by moollaza** [comment]
@Juholei overall this looks pretty great, thanks for an awesome contribution!

File: share/goodie/cheat_sheets/svn.json (hunk 1 above)
Comment (Juholei) [inline share/goodie/cheat_sheets/svn.json]: I updated the description to be more helpful. I hope it is ok now.

**Comment by Juholei** [comment]
Description updated.

**Comment by moollaza** [comment]
@Juholei thanks for updating!

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@Juholei congrats! It's live: https://duckduckgo.com/?q=svn+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**